internal/lexer/gen/golang/lexer.go

```diff
@@ -104,32 +104,32 @@ func NewLexerFile(fpath string) (*Lexer, error) {
 	return NewLexer(src), nil
 }
 
-func (this *Lexer) Scan() (tok *token.Token) {
+func (l *Lexer) Scan() (tok *token.Token) {
 	{{- if .Debug}}
-	fmt.Printf("Lexer.Scan() pos=%d\n", this.pos)
+	fmt.Printf("Lexer.Scan() pos=%d\n", l.pos)
 	{{- else}}
-	// fmt.Printf("Lexer.Scan() pos=%d\n", this.pos)
+	// fmt.Printf("Lexer.Scan() pos=%d\n", l.pos)
 	{{- end}}
 	tok = new(token.Token)
-	if this.pos >= len(this.src) {
+	if l.pos >= len(l.src) {
 		tok.Type = token.EOF
-		tok.Pos.Offset, tok.Pos.Line, tok.Pos.Column = this.pos, this.line, this.column
+		tok.Pos.Offset, tok.Pos.Line, tok.Pos.Column = l.pos, l.line, l.column
 		return
 	}
-	start, startLine, startColumn, end := this.pos, this.line, this.column, 0
+	start, startLine, startColumn, end := l.pos, l.line, l.column, 0
 	tok.Type = token.INVALID
 	state, rune1, size := 0, rune(-1), 0
 	for state != -1 {
 	{{- if .Debug}}
-		fmt.Printf("\tpos=%d, line=%d, col=%d, state=%d\n", this.pos, this.line, this.column, state)
+		fmt.Printf("\tpos=%d, line=%d, col=%d, state=%d\n", l.pos, l.line, l.column, state)
 	{{- else}}
-		// fmt.Printf("\tpos=%d, line=%d, col=%d, state=%d\n", this.pos, this.line, this.column, state)
+		// fmt.Printf("\tpos=%d, line=%d, col=%d, state=%d\n", l.pos, l.line, l.column, state)
```

Comment:
These were quite useful in debugging the false line column reporting on backtrace, but that is fixed. They might be useful for creating functionality for a better error message, but should probably be temporarily added more customized when creating that functionality (I haven't gotten around to it). I do not think there is any reason to have them in production code. But i might be wrong.